daml-lf/transaction/src/main/protobuf/com/digitalasset/daml/lf/transaction.proto

```diff
@@ -11,7 +11,8 @@
 // * 4: string contract_id replaced globally by ContractId message
 // * 5: new field actors in NodeFetch
 // * 6: removal of controllers in exercise nodes
-// * 7: new field exerciseResult in NodeExercise
+// * 7: new field return_value in NodeExercise
+// * 8: new field contract_key in NodeExercise
```

Comment:
Please update `daml-lf/spec/transaction.rst` describing the new semantics as well:

* [ ] version at top of file
* [ ] entry in version history table
* [ ] definition of the field at the end of the message NodeExercise section

Reply:
updated.